src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4680,4 +4680,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_PeWritingFailure" xml:space="preserve">
     <value>An error occurred while writing the output file: {0}</value>
   </data>
+  <data name="WRN_AttributeIgnoredWhenPublicSigning" xml:space="preserve">
+    <value>Attribute '{0}' is ignored when public signing is specified.</value>
+  </data>
+  <data name="WRN_AttributeIgnoredWhenPublicSigning_Title" xml:space="preserve">
```

Comment:
I have no idea what that test is doing... When you move the ErrorCodes to the bottom, does that test still fail?

---
In reply to: [63416950](https://github.com/dotnet/roslyn/pull/11334#discussion_r63416950) [](ancestors = 63416950)